plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/managers/BuildProgressSplitterPanelManager.kt

```diff
@@ -176,8 +176,9 @@ class BuildProgressSplitterPanelManager(private val project: Project) :
         } else {
             if (backendProgressStepsAvailable && transformationPlan != null) {
                 currentBuildingTransformationStep = newBuildingTransformationStep
-                newBuildingTransformationStep = transformationPlan.transformationSteps().size
-                val transformationPlanSteps = transformationPlan.transformationSteps()?.map {
+                // skip step 0 (contains supplemental info)
+                newBuildingTransformationStep = transformationPlan.transformationSteps().size - 1 // TO-DO: what to set this to?
```

Comment:
What is this variable doing? Wasn't clear to me.

Reply:
Since we added a step # 0 to the GetPlan response, I thought we need to offset this by + or - 1?